Composer/packages/ui-plugins/select-skill-dialog/src/ComboBoxField.tsx

```diff
@@ -3,7 +3,6 @@
 
```

Comment:
Can this component move into the form library? I think we would need to support the `widget` ui option first.

Reply:
This component has some logic specific to the plugin, so I don't know if moving it to the component library makes sense.